guides/v2.1/get-started/authentication/gs-authentication.md

```diff
@@ -34,7 +34,7 @@ Guest user | Resources with `anonymous` permission.
 
 <h3 id="acl-webapi-relation">Relation between acl.xml and webapi.xml</h3>
 The <code>acl.xml</code> file defines the access control list (ACL) for a given {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %}. It defines available set of permissions to access the resources.
-`acl.xml` files across all Magento modules are consolidated to build an ACL tree which is used to select allowed {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %} role resources or third party Integration's access (System > {% glossarytooltip 55774db9-bf9d-40f3-83db-b10cc5ae3b68 %}Extension{% endglossarytooltip %} > Integration > Add New Integration > Available APIs).
+`acl.xml` files across all Magento modules are consolidated to build an ACL tree which is used to select allowed {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %} role resources or third party Integration's access (<b>System > Extension > Integration > Add New Integration > Available APIs</b>).
```

Comment:
We use markdown for text formatting. Please replace `<b>` with `**` surrounding what you want to be bold.

Reply:
Any other places that you can convert HTML formatting to markdown formatting is also good!